web/source/resources/osk/kmwosk.css

```diff
@@ -430,6 +428,61 @@ div.android div.kmw-keytip-cap {
   }
 }
 
+/*** TODO:  extract to a separate CSS file used only in embedded mode! ***/
```

Comment:
Don't forget this TODO

Reply:
Right.  I've yet to do this in part because it _does_ mean linking an extra file, which requires proper pathing based on the `init`'s `"resources"` parameter, and that may take a moment or two to sort out.  I'll probably be giving this its own commit.